tests/testthat/test-pmcmc.R

```diff
@@ -66,14 +73,36 @@ test_that("pmcmc runs without error", {
  expect_equal(object = Z$results$start_date, 
               expected = rep(Z$inputs$pars$pars_init$start_date, n_mcmc + 1))
  expect_true(!all(diff(Z$results$log_likelihood) == 0))
+
  
-  
+set.seed(1)
+
+X2 <- pmcmc(
+  data = data, 
+  n_mcmc = n_mcmc,
+  sircovid_model = sircovid_model,
+  pars_obs = list(phi_ICU = 0.95,
+                  k_ICU = 2,
+                  phi_death = 926 / 1019,
+                  k_death = 2,
+                  exp_noise = 1e6),
+  cov_mat = matrix(c(0.001^2, 0,
```

Comment:
Can you add a test which has non-zero covariance between proposals just to check it runs. I don't think there's any  need to check exact run results, but maybe that you get different results from `X2` with the same seed would be sensible